lib/Solver/STPSolver.h

```diff
@@ -30,9 +30,8 @@ class STPSolver : public Solver {
   std::string getConstraintLog(const Query &) override;
 
   /// setCoreSolverTimeout - Set constraint solver timeout delay to the given
-  /// value; 0
-  /// is off.
-  virtual void setCoreSolverTimeout(time::Span timeout);
+  /// value; 0 is off.
+  virtual void setCoreSolverTimeout(time::Span timeout) override;
```

Comment:
If a function is marked `override`, there is no point to also mark it `virtual`.